src/Uniplaces/Phpunit/ScrutinizerCloverListener.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Uniplaces\Phpunit;
+
+use Symfony\Component\Process\Process;
+
+/**
+ * Class ScrutinizerCloverListener
+ * @package Uniplaces
+ */
+class ScrutinizerCloverListener extends \PHPUnit_Framework_BaseTestListener
+{
+
+    const OCULAR_PHAR = 'ocular.phar';
+    const OCULAR_HOST = 'https://scrutinizer-ci.com';
+    const INSTALL_DIR = '/tmp';
+    const SCRUTINIZER_API_TOKEN = '7c8ad0ef7f0a42323cee76eaaf9da7a7339acd24b23ccd09a1dbe2f1c5fbcaa9';
+
+    /**
+     * @param \PHPUnit_Framework_TestSuite $suite
+     *
+     * @return bool
+     * @throws \Exception
+     */
+    public function endTestSuite(\PHPUnit_Framework_TestSuite $suite)
+    {
+        $file = self::INSTALL_DIR . DIRECTORY_SEPARATOR  . self::OCULAR_PHAR;
+
+        if (!is_file($file)) {
+            if (!$this->downloadOcular()) {
+                throw new \Exception('Ocular could not be downloaded');
+            }
+        }
+
+        if (!$this->uploadCloverReport()) {
+            throw new \Exception('Clover report could not be uploaded');
+        }
+
+        return true;
+    }
+
+    /**
+     * @return bool
+     */
+    protected function uploadCloverReport()
+    {
+        $process ='php ' . self::INSTALL_DIR . DIRECTORY_SEPARATOR . self::OCULAR_PHAR . ' code-coverage:upload --access-token="'
+            . self::SCRUTINIZER_API_TOKEN . '" --format=php-clover ' . __DIR__ . '/../../docs/phpunit/clover.xml';
```

Comment:
clover report file is statically defined. will provide a more dynamic way for this